.github/workflows/ansible-test.yml

```diff
@@ -183,15 +125,14 @@ jobs:
     needs:
       - sanity
       - units
-      - integration
 
     runs-on: ubuntu-latest
 
     steps:
       - name: Decide whether the needed jobs succeeded or failed
         uses: re-actors/alls-green@release/v1
         with:
-          allowed-skips: integration, units  # TODO: drop once enabled
+          allowed-skips: integration
```

Comment:
@samdoran there's no need to have a job name that doesn't exist. You could've just deleted the entire input.

Reply:
Good point. I'll clean that up.